src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -108,17 +106,17 @@ public ActivityLogEntry(AppLogLine appLog) {
      * Used in the various servlets in the application
      */
     public ActivityLogEntry(String servlet, String act, AccountAttributes acc, String params, String link) {
-        this(servlet, act, acc, params, link, null, null);
+        this(servlet, act, acc, params, link, null, null, null);
```

Comment:
This is not correct. Why is the last parameter `null`? In the old code, the `UserType` parameter is obtained from `GateKeeper`, which in most cases is not a `null`.
Also applies to two other instances down there where `null` parameter is passed.